src/Tribe/Templates.php

```diff
@@ -16,8 +16,8 @@ class Tribe__Tickets__Templates extends Tribe__Templates {
 	 **/
 	public static function get_template_hierarchy( $template, $args = array() ) {
 		if ( ! is_array( $args ) ) {
+			$passed        = [ $template, $args ];
```

Comment:
This doesn't seem backcompat with the `$backwards_map`, am I wrong?

Reply:
Original TEC PR has the same backcompat breakage: https://github.com/moderntribe/the-events-calendar/pull/2750/files

Reply:
```suggestion
			$passed        = func_get_args();
```